client/src/components/App.jsx

```diff
@@ -15,14 +15,38 @@ class App extends React.Component {
   }
 
   componentDidMount() {
+    $.ajax({
+      url: `/api/rooms/${this.props.roomId}/neighborhood`,
+      // dataType: 'json',
+      method: 'GET',
+      // contentType: 'application/json',
+      // data: JSON.stringify({jsonData: data}),
+      success: (data) => {
+        console.log(data);
+        this.setState({
+          hostName: data[0].name,
+          city: data[0].city,
+          country: data[0].country,
+          gettingAround: data[0].gettingAround,
+          description: data[0].description,
+          guidebookId: data[0].guidebookId,
```

Comment:
I would consider making a variable, such as `neighborhood` here. I know you want to use `nb`, but if someone who wasn't us was reading this, they wouldn't know what you were talking about and would be confused.